### ISSUE TITLE
InflateException: Binary XML file line #15: Binary XML file line #15: Error inflating class androidx.appcompat.widg...

### ISSUE THREAD

**Issue by pocmo** [mozilla-mobile/fenix#4485]
https://sentry.prod.mozaws.net/operations/fenix-nightly/issues/6110179/

```
InflateException: Binary XML file line #15: Binary XML file line #15: Error inflating class androidx.appcompat.widget.AppCompatTextView
    at org.mozilla.fenix.home.sessioncontrol.SessionControlAdapter.onCreateViewHolder(SessionControlAdapter.kt:1)
...
```

```
XmlPullParserException: Binary XML file line #10: invalid color state list tag gradient
    at android.content.res.ColorStateList.createFromXmlInner(ColorStateList.java:217)
    at android.content.res.ColorStateList.createFromXml(ColorStateList.java:201)
    at android.content.res.Resources.loadColorStateListForCookie(Resources.java:2746)
    at android.content.res.Resources.loadColorStateList(Resources.java:2699)
    at android.content.res.TypedArray.getColor(TypedArray.java:439)
...
(87 additional frame(s) were not displayed)

NotFoundException: File res/drawable/$ic_onboarding_firefox_accounts__0.xml from color state list resource ID #0x7f08000d
    at android.content.res.Resources.loadColorStateListForCookie(Resources.java:2750)
    at android.content.res.Resources.loadColorStateList(Resources.java:2699)
    at android.content.res.TypedArray.getColor(TypedArray.java:439)
    at android.graphics.drawable.VectorDrawable$VFullPath.updateStateFromTypedArray(VectorDrawable.java:1605)
    at android.graphics.drawable.VectorDrawable$VFullPath.inflate(VectorDrawable.java:1584)
...
(85 additional frame(s) were not displayed)

NotFoundException: File res/drawable/ic_onboarding_firefox_accounts.xml from drawable resource ID #0x7f0800ee
    at android.content.res.Resources.loadDrawableForCookie(Resources.java:2640)
    at android.content.res.Resources.loadDrawable(Resources.java:2540)
    at android.content.res.TypedArray.getDrawable(TypedArray.java:870)
    at android.widget.TextView.<init>(TextView.java:897)
    at android.widget.TextView.<init>(TextView.java:671)
...
(76 additional frame(s) were not displayed)

InvocationTargetException: None
    at java.lang.reflect.Constructor.newInstance(Constructor.java)
    at android.view.LayoutInflater.createView(LayoutInflater.java:619)
    at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:764)
    at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:704)
    at android.view.LayoutInflater.rInflate(LayoutInflater.java:835)
...
(69 additional frame(s) were not displayed)

InflateException: Binary XML file line #15: Error inflating class androidx.appcompat.widget.AppCompatTextView
    at android.view.LayoutInflater.createView(LayoutInflater.java:645)
    at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:764)
    at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:704)
    at android.view.LayoutInflater.rInflate(LayoutInflater.java:835)
    at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:798)
...
(68 additional frame(s) were not displayed)

InflateException: Binary XML file line #15: Binary XML file line #15: Error inflating class androidx.appcompat.widget.AppCompatTextView
    at org.mozilla.fenix.home.sessioncontrol.SessionControlAdapter.onCreateViewHolder(SessionControlAdapter.kt:1)
...
(63 additional frame(s) were not displayed)

Binary XML file line #15: Binary XML file line #15: Error inflating class androidx.appcompat.widget.AppCompatTextView
```

**Comment by pocmo** [comment]
This seems to happen frequently on a Nexus 6, API 21 (Firebase) and in our pre-launch reports (Android 6.0 devices). Looks like some issue on Android 5-6?

**Comment by ekager** [comment]
I was able to reproduce this locally on older devices. They don't support gradients in their vector files to be set in XML? I think this should be fixed by https://github.com/mozilla-mobile/fenix/pull/4523 , but we should keep an eye on Sentry.

**Comment by ekager** [comment]
Closing because this should be fixed by #4523
